packages/ai-openai/src/node/openai-streaming-iterator.ts

```diff
@@ -34,7 +34,7 @@ export class StreamingAsyncIterator implements AsyncIterableIterator<LanguageMod
             this.dispose();
         });
         this.registerStreamListener('abort', () => {
-            console.log('Got an abort!');
+            console.debug('Got an abort!');
```

Comment:
Whoops, I'll just remove this - at once point I'd forgotten to pass in the `CancellationToken` and I was confused about why nothing was ever cancelling.